pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -198,10 +198,12 @@ class Measurements final
         case ExpValFunc::Hadamard:
             return applyExpValNamedFunctor<getExpectationValueHadamardFunctor,
                                            1>(wires);
+            // LCOV_EXCL_START
```

Comment:
We are also excluding this part of the code from coverage. Is there something we can do to include it? Are you planning a follow-up PR?